src/TSMapEditor/Config/Constants.ini

```
@@ -50,6 +50,9 @@ EnableIniInclude=false
 ; Should the editor consider Phobos $Inherits entries?
 EnableIniInheritance=false
 
+; Should local and global variables be integer values (as opposed to true/false booleans)?
+IntegerVariables=false;
```

```suggestion
IntegerVariables=false
```

Iow. remove the semicolon from the end of the line (it's INI code, not a C-derived language)